be/src/connector/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# This file is made available under Elastic License 2.0.
```

Comment:
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.